guides/source/getting_started.md

```diff
@@ -1966,7 +1968,7 @@ class CommentsController < ApplicationController
 
   private
     def comment_params
-      params.require(:comment).permit(:commenter, :body)
+      params.require(:comment).permit(:commenter, :body, :status)
```

Comment:
[Section 8.3](https://guides.rubyonrails.org/getting_started.html#using-concerns) later asks the developer to write the following in `app/controllers/comments_controller.rb`:

```ruby
private
    def comment_params
      params.require(:comment).permit(:commenter, :body, :status)
    end
```